src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -573,6 +576,18 @@ def transform_request(self,
 
         return final_response
 
+    def _generate_manifest_uuid_name(self, filters: JSON) -> str:
+        """
+        Generate and return a UUID string generated using the latest git commit and filters
+        :param filters: The filters used in the manifest request
+        :return: String representation of a UUID
+        """
+        uuid_namespace = uuid.UUID('ca1df635-b42c-4671-9322-b0a7209f0235')
+        git_status = config._git_status
```

Comment:
There is a reason this property is protected. This would fail when run in a lambda.

Use lambda_git_status instead and mock as necessary during the test.